tests/test_validation.py

```diff
@@ -1083,3 +1054,920 @@ def test_login(self, dmge: DataModelGraphExplorer) -> None:
             validate_attribute.synStore.fileview_query
             == "SELECT name,id,path FROM syn23643253 WHERE parentId='syn61682648' AND type='file' AND projectId IN ('syn23643250', '') ;"
         )
+
+    def test__get_target_manifest_dataframes(self, va_obj: ValidateAttribute) -> None:
+        """Testing for ValidateAttribute._get_target_manifest_dataframes"""
+        manifests = va_obj._get_target_manifest_dataframes(
+            "patient", project_scope=["syn54126707"]
+        )
+        assert list(manifests.keys()) == ["syn54126997", "syn54127001"]
+
+
+@pytest.fixture(name="cross_val_df1")
+def fixture_cross_val_df1() -> Generator[DataFrame, None, None]:
+    """Yields a dataframe"""
+    df = DataFrame(
+        {
+            "PatientID": ["A", "B", "C"],
+            "component": ["comp1", "comp1", "comp1"],
+            "id": ["id1", "id2", "id3"],
+            "entityid": ["x", "x", "x"],
+        }
+    )
+    yield df
+
+
+@pytest.fixture(name="cross_val_df2")
+def fixture_cross_val_df2(cross_val_df1: DataFrame) -> Generator[DataFrame, None, None]:
+    """Yields dataframe df1 with an extra row"""
+    df = concat(
+        [
+            cross_val_df1,
+            DataFrame(
+                {
+                    "PatientID": ["D"],
+                    "component": ["comp1"],
+                    "id": ["id4"],
+                    "entityid": ["x"],
+                }
+            ),
+        ]
+    )
+    yield df
+
+
+@pytest.fixture(name="cross_val_df3")
+def fixture_cross_val_df3() -> Generator[DataFrame, None, None]:
+    """Yields empty dataframe"""
+    df = DataFrame(
+        {
+            "PatientID": [],
+            "component": [],
+            "id": [],
+            "entityid": [],
+        }
+    )
+    yield df
+
+
+@pytest.fixture(name="cross_val_col_names")
+def fixture_cross_val_col_names() -> Generator[dict[str, str], None, None]:
+    """
+    Yields:
+        Generator[dict[str, str], None, None]: A dicitonary of column names
+          keys are the label, and values are the display name
+    """
+    column_names = {
+        "patientid": "PatientID",
+        "component": "component",
+        "id": "id",
+        "entityid": "entityid",
+    }
+    yield column_names
+
+
+@pytest.fixture(name="va_obj")
+def fixture_va_obj(
+    dmge: DataModelGraphExplorer,
+) -> Generator[ValidateAttribute, None, None]:
+    """Yield a ValidateAttribute object"""
+    yield ValidateAttribute(dmge)
+
+
+class TestUnitValidateAttributeObject:
```

Comment:
If all of the calls are mocked here this might be a great first candidate to move over to `tests/unit/test_{name}.py`.

I started the process of putting my new tests into the similarly named `tests/integration` directory.

Reply:
Done!